devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -640,7 +643,8 @@ void prepare() throws Exception {
             }
             args.add("-jar");
             args.add(tempFile.getAbsolutePath());
-
+            if (argsString != null)
+                args.addAll(Arrays.asList(argsString.split(" ")));
```

Comment:
This does not really deal with escaping, but I guess we can handle that later.

Reply:
I have dug up the correct way to escape from the JavaExecMojo.
Should be properly escaped now.